frontend/src/container/QueryBuilder/components/AdditionalFiltersToggler/AdditionalFiltersToggler.tsx

```diff
@@ -14,20 +14,23 @@ import {
 export const AdditionalFiltersToggler = memo(function AdditionalFiltersToggler({
 	children,
 	listOfAdditionalFilter,
+	hideLimit,
+	queryname,
 }: AdditionalFiltersProps): JSX.Element {
 	const [isOpenedFilters, setIsOpenedFilters] = useState<boolean>(false);
 
 	const handleToggleOpenFilters = (): void => {
 		setIsOpenedFilters((prevState) => !prevState);
 	};
-
-	const filtersTexts: ReactNode = listOfAdditionalFilter.map((str, index) => {
-		const isNextLast = index + 1 === listOfAdditionalFilter.length - 1;
-
-		if (index === listOfAdditionalFilter.length - 1) {
+	const filtersToDisplay = listOfAdditionalFilter.filter(
```

Comment:
we have filterConfig itself in the query builder not to render the filters

Reply:
@palashgdev As per your suggestion the code has been refactored because of filterConfig for removing Limit,but for the text Limit which is present inside AdditionalFiltersToggler I had to filter limit for list.Kindly review the PR.

Reply:
@palashgdev PR updated